utility/utils.py

```diff
@@ -662,3 +665,55 @@ def download_file(url, filename):
         r = requests.get(url)
         f.write(r.content)
     assert r.ok
+
+
+def destroy_cluster(cluster_path):
```

Comment:
what about to have it as another test case? Then we can skip it depends on some parameter?

Reply:
What do we benefit from it being it's own test case? I can see that complicating how we call it when we use the CLI usage pattern, as well as adding more than just a single line to a suite file in order to destroy the cluster after a certain test.

Reply:
ok, I see now when I looked at PR with fresh brain after night.

Reply:
Thinking about this again, it really doesn't change how we call it from the CLI side. The test case however will basically just end up calling the function in the same way.